utils/python/CIME/case_run.py

```diff
@@ -171,7 +171,8 @@ def resubmit_check(case):
     elif dout_s and mach == 'mira':
         caseroot = case.get_value("CASEROOT")
         cimeroot = case.get_value("CIMEROOT")
-        cmd = "ssh cooleylogin1 'cd %s; CIMEROOT=%s ./case.submit %s --job case.st_archive' "%(caseroot, cimeroot, caseroot)
+        cmd = "ssh cooleylogin1 'cd %s; CIMEROOT=%s ./case.submit %s"%(caseroot, cimeroot, caseroot)
+        cmd = cmd + " --job case.st_archive' "
```

Comment:
I would encourage you to use scripts/Tool/code_checker as it has all the CIME-agreed-upon pylint settings. It will scan all files by default but you can just provide "case_run.py" as an argument to limit the scan. You can run in debug mode to see the pylint output.